flow/Makefile

```diff
@@ -844,35 +855,51 @@ clean_abstract:
 
 # Merge wrapped macros using Klayout
 #-------------------------------------------------------------------------------
-$(WRAPPED_GDSOAS): $(OBJECTS_DIR)/klayout_wrap.lyt $(WRAPPED_LEFS)
+$(WRAPPED_GDSOAS) &:: $(OBJECTS_DIR)/klayout_wrap.lyt $(WRAPPED_LEFS)
+	$(UNSET_AND_MAKE) do-wrapped-gdsoas
+
+.PHONY: do-wrapped-gdsoas
+do-wrapped-gdsoas:
 	$(call KLAYOUT_FOUND)
-	($(TIME_CMD) $(KLAYOUT_CMD) -zz -rd design_name=$(basename $(notdir $@)) \
-	        -rd in_def=$(OBJECTS_DIR)/def/$(notdir $(@:$(STREAM_SYSTEM_EXT)=def)) \
-	        -rd in_files="$(ADDITIONAL_GDSOAS)" \
-	        -rd config_file=$(FILL_CONFIG) \
-	        -rd seal_file="" \
-	        -rd out_file=$@ \
-	        -rd tech_file=$(OBJECTS_DIR)/klayout_wrap.lyt \
-	        -rd layer_map=$(GDS_LAYER_MAP) \
-	        -r $(UTILS_DIR)/def2stream.py) 2>&1 | tee $(LOG_DIR)/6_merge_$(basename $(notdir $@)).log
+	for target in $(WRAPPED_GDSOAS); do \
+		($(TIME_CMD) $(KLAYOUT_CMD) -zz \
+		-rd design_name=$(basename $(notdir $$target)) \
+		-rd in_def=$(OBJECTS_DIR)/def/$(notdir $${target:$(STREAM_SYSTEM_EXT)=def}) \
```

Comment:
I tried to test this w/`make DESIGN_CONFIG=designs/gf12/bp_dual/config.mk do-wrapped-gdsoas` and modifying designs/gf12/bp_dual/config.mk to use asap7. Obviously that testing was quite limited as it failed, but I was able to fix some basic syntax errors.

@maliberty Could one of the non-NDA PDKs be modified to use WRAPPED_GDSOAS just so as to make testing easier?

Reply:
The implementation was made more complicated by the fact that centos 7 is using make prior to 4.3. In 4.3, "&::" is supported, which would have made the implementation cleaner.

Reply:
@maliberty I'm removing support for WRAP_LEFS.

Reply:
Why?

Reply:
I'm removing support for WRAP_LEFS for the do-gds case. For normal builds it is still there unchanged.

If someone wants do-gds for gf12, they either have to write support WRAP_LEFS or pay back the technical debt so that WRAP_LEFS is no longer needed.